### PR TITLE
feature/configure dependabot

### LINKED ISSUE #78: Remediation of High Severity Vulnerability
Our primary task involves fixing a high severity vulnerability by updating our Vite to its current version, and then identify any other resources that require immediate updates.

### LINKED ISSUE #103: Configure dependabot

### PR DESCRIPTION
**Pull request by Quiddlee**
## What type of PR is this? (select all that apply)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description
Configure dependabot
## Related Tickets & Documents

- Related Issue #78 
- Closes #78
- Related Issue #103  
- Closes #103

## Screenshots, Recordings
image

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

**Review by andron13 — APPROVED** [review]
good

**Review by ramzes-84 — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    assignees:
```
Comment (SpaNb4) [inline .github/dependabot.yml:7]: Why not `reviewers`?

File: .github/dependabot.yml
Comment (SpaNb4) [inline .github/dependabot.yml:1]: I'd add this option as well https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

File: .github/dependabot.yml (hunk 1 above)
Comment (Quiddlee) [reply .github/dependabot.yml:7]: Fixed

File: .github/dependabot.yml
Comment (SpaNb4) [reply .github/dependabot.yml:1]: I didn't know that it opens a maximum of 5 PRs by default, so we're good

**Review by SpaNb4 — APPROVED** [review]

**Pull request merged**